documentation/release_notes_6.markdown

```diff
@@ -17,6 +17,35 @@ canonical: "/puppetdb/latest/release_notes.html"
 
 # PuppetDB: Release notes
 
+## PuppetDB 6.22.0
+
+Released July 26 2022
```

Comment:
Looks like August 2 is the new date? according to the #proj-puppet-releases channel